### ISSUE TITLE
Selected video format is not supported loop in 11.62

### ISSUE THREAD

**Issue by rcarter515** [yuliskov/SmartTubeNext#760]
One of these days I'm going to find an old version download and keep it handy for these situations.  Hasn't been stable since 11.58 with it's on quirks.

**Comment by borislip** [comment]
Same here on Sony Bravia 2015, unlike vp9, which runs at slideshow frame rate, av01 doesn't seem to be supported by this TV at all. And yhr app just keeps saying it in a loop instead of falling bavk to avc.

Why can't we just have an option to enable/disable specific codecs?

**Comment by stodd3** [comment]
Updated to 11.63 and error message is occurring still. Was fine in 11.62

**Comment by dlorde** [comment]
Yeah, I get this with 11.64 - on some channels (e.g. Marques Brownlee), half the videos play OK, but half give the 'Selected video format not supported' loop.

**Comment by stodd3** [comment]
Still happens in 11.64, worked fine for several videos and then "Selected video format is not supported", even if I just played the video a couple of minutes before with no issue.

Tried  changing to another option in the Video Format list, none worked.

**Comment by dlorde** [comment]
11.65 seems to have fixed the 'Selected video format not supported' loop issue for me.

**Comment by USvsTHEM** [comment]
but they update so much and with Google always doin spiteful shit like a scorn woman the next update will break it again i promise you it aint the dev team with Y its those weirdos at Google
